engine/lib/phx/script/ffi_gen/Engine.lua

```diff
@@ -53,11 +53,17 @@ function Loader.defineType()
         local t  = ffi.typeof('Engine')
         local mt = {
             __index = {
-                window  = libphx.Engine_Window,
-                input   = libphx.Engine_Input,
-                hmGui   = libphx.Engine_HmGui,
-                getTime = libphx.Engine_GetTime,
-                exit    = libphx.Engine_Exit,
+                window     = libphx.Engine_Window,
```

Comment:
How did you get these changes? This file is auto generated from the Rust code and there are no changes in the engine in this commit.

Reply:
completely forgot that we autogen 🤦🏽‍♂️ - check if this is fine now

Reply:
Do we have other way to do this except of adding `self` to `Engine::get_version()`?
It feels like workaround. @dgavedissian what do you think?

Reply:
Yeah, I'm not sure why we added `self` to `get_version` here. The return value of `env!` is already a `&'static str` afaik

Reply:
I don´t know my way around the auto FFI gen, is there another way of getting it to generate the needed getVersion metatype?